src/frontend/Sider/Behandling/Vilkårvurdering/VurderDelvilkår.tsx

```diff
@@ -28,7 +33,7 @@ const VurderDelvilkår: FC<Props> = ({ regel, vurdering, settVurdering }) => {
                             })
                         }
                     >
-                        {svarId}
+                        {svarIdTilTekst[svarId] || svarId}
```

Comment:
Tror aldri høyresiden her vil brukes 🤔  Enten vil den finne tekst for svarId, eller så vil den gi en runtimefeil

Reply:
Hvis den ikke finnes vises ikke noe. Så denne løsningen er egentlig mer en løsning for å gjøre det smudere for oss å finne ut hvilket svar som faktisk ikke har en tekst. Hvis vi ikke ser noe så må vi sammenligne alle svarID-er i frontend og backend for å finne ut hvilken som mangler. 

Dette er en case hver gang man legger til en regel med annet enn "ja/nei" svar.

Kan fjerne men da vises ikke noe om man ikke har lagt inn teksten

Reply:
Ja, jeg tenkte litt feil her. Tenkte at den ville gi en typeerror på `svarTiltekst["nøkkelSomIkkeFinnes"]` men gir jo bare undefined